src/routes/Admin/AdminView.js

```diff
@@ -13,31 +18,171 @@ class Admin extends Component {
     this.state = {
       products: [],
       isLading: false,
+      showModal: false,
+      showingLoadForms: false,
     };
-    this.clickItem = this.clickItem.bind(this);
+
+    this.navigateToItem = this.navigateToItem.bind(this);
+    this.handleEdit = this.handleEdit.bind(this);
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+    this.handleCloseModalQuick = this.handleCloseModalQuick.bind(this);
+    this.deleteItem = this.deleteItem.bind(this);
+    this.createAddItemGlobal = this.createAddItemGlobal.bind(this);
+    this.modalCreate = this.modalCreate.bind(this);
   }
 
   async componentDidMount() {
     this.setState({ isLading: true });
```

Comment:
you can set the `isLoading` state param to the `true` in your constructor instead of triggering yet another unnecessary render